bears/general/FilenameBear.py

```diff
@@ -63,6 +71,8 @@ def run(self, filename, file,
                           'Using the default "snake" naming convention.')
                 file_naming_convention = 'snake'
 
+        messages = []
```

Comment:
I don't like the idea of collecting all the errors into one diff, what if I want to apply the patch that renames it to follow naming convention, but not the patch that renames it with prefix text :confused: ?